### PR TITLE
[Chatting Server] Project Init

### PR DESCRIPTION
**Pull request by rian-yeji**

### CODE REVIEW

**Comment by Junghun0** [comment]
확인했습니다~

**Comment by jnsorn** [comment]
좋아용

File: src/BACKEND/chatting/.mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    private static final String MAVEN_WRAPPER_JAR_PATH =
+            ".mvn/wrapper/maven-wrapper.jar";
+
+    /**
+     * Name of the property which should be used to override the default download url for the wrapper.
+     */
+    private static final String PROPERTY_NAME_WRAPPER_URL = "wrapperUrl";
+
+    public static void main(String args[]) {
+        System.out.println("- Downloader started");
```
Comment (dydtjr1128) [inline src/BACKEND/chatting/.mvn/wrapper/MavenWrapperDownloader.java]: Change log to logback

File: src/BACKEND/chatting/.mvn/wrapper/MavenWrapperDownloader.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                    }
+                } catch (IOException e) {
+                    // Ignore ...
+                }
+            }
+        }
+        System.out.println("- Downloading from: : " + url);
+
+        File outputFile = new File(baseDirectory.getAbsolutePath(), MAVEN_WRAPPER_JAR_PATH);
+        if (!outputFile.getParentFile().exists()) {
```
Comment (zion3mx) [inline src/BACKEND/chatting/.mvn/wrapper/MavenWrapperDownloader.java]: The code below is good to read.
```
 if (outputFile.getParentFile().exists()  || outputFile.getParentFile().mkdirs()) {
 throw new .....
}
 
```

**Comment by haaaaaneul** [comment]
확인했어요~

File: src/BACKEND/chatting/.mvn/wrapper/MavenWrapperDownloader.java (hunk 2 above)
Comment (rian-yeji) [reply src/BACKEND/chatting/.mvn/wrapper/MavenWrapperDownloader.java]: throw에서 File(baseDirectory.getAbsolutePath(), MAVEN_WRAPPER_JAR_PATH)를 다시 하는건가요?

**Pull request merged**